src/frontend/Page/Docs.elm

```diff
@@ -561,24 +655,206 @@ viewValueItem { author, project, version } moduleName ownerName valueName =
     url =
       Href.toModule author project version moduleName (Just ownerName)
   in
-  li [ class "pkg-nav-value" ] [ navLink valueName url False ]
+  li [ class "pkg-nav-value" ] [ navModuleLink valueName url False ]
+
+
+
+-- VIEW ABOUT
+
+
+viewAbout : Status Project.PackageInfo -> Status Time.Posix -> Html msg
+viewAbout manifestStatus time =
+  case manifestStatus of
+    Success manifest ->
+      div [ class "block-list pkg-about" ]
+        [ h1 [ class "block-list-title" ] [ text "About" ]
+        , text manifest.summary
+        , viewInstall manifest
+        , viewRelease manifest time
+        , viewLicense manifest
+        , viewDependencies manifest
+        ]
+
+    Loading ->
+      div [ class "block-list pkg-about" ] [ text "" ] -- TODO
+
+    Failure ->
+      div
+        (class "block-list pkg-about" :: Problem.styles)
+        (Problem.offline "elm.json")
+
+
+viewInstall : Project.PackageInfo -> Html msg
+viewInstall manifest =
+  if requiresElmCore manifest.deps then
+    -- >= 0.19
+    viewInstallHelp "elm install" (Package.toString manifest.name)
+
+  else if requiresElmLangCore manifest.deps then
+    -- < 0.19.0
+    viewInstallHelp "elm-package install" (Package.toString manifest.name)
```

Comment:
Is this command right for all versions from 0.14 to 0.18?

Reply:
At some point it was called `elm-get`. I do not recall when it was renamed.